pipeline/restart_node.sh

```diff
@@ -441,6 +441,16 @@ wait_for_harmony_process_to_exit() {
 	done
 }
 
+function cleanup_exp_db
+{
+	# clean up explorer db if exist
+	exp_db="/home/ec2-user/explorer_storage_*"
```

Comment:
no need to hardcode /home/ec2-user, just use the current directory, so that it will work on GCP as well.